### DISCUSSION TITLE
Fork a repository as a starting point for my idea :)

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [community/community#13326] [Pull Requests] [answered]
Hey there,

I'm not quite sure how to explain this clearly, but I'll try my best and try to not sound bad :)

In short, the person who maintained this [repository](https://github.com/leereilly/games), since 2018, no longer makes any changes (at least when I'm posting this). And I thought I should then create a fork and start [my own separate version](https://github.com/201flaviosilva/Open-Source-Games) so that the community doesn't get stuck and without updates.

**I have no intention of making any pull requests to the original repository, but whenever I create a branch and then do a PR, it automatically points to the original repository. My question is can you change this by default? How? (I ask this because I imagine a more inattentive contributor (like me) might still do a PR to the original repository by mistake.)**

**My other question is, how do I make contributions from this repository appear in the GitHub user's contributions graph?
(This question is not very important)**

And by the way, some tips, settings, etc... new projects based on others? Like this one :)

Final notes, my intention is not to steal work from the original repository, and I just want to continue the work that has currently been stalled for almost 4 years.

(Sorry if I was confused, and if this is not the right category 😅)

**Comment by airtower-luna** [comment, marked as answer]
> My other question is, how do I make contributions from this repository appear in the GitHub user's contributions graph?

You can contact support to [detach the fork](https://support.github.com/request/fork), so it becomes an independent repository. That'll also solve the PR problem. :wink: 

> Final notes, my intention is not to steal work from the original repository, and I just want to continue the work that has currently been stalled for almost 4 years.

For that you'll have to follow the license of the original repository, what that means depends on the exact license. Being able to take code and make your own improvements is one of the core ideas of Open Source. :smiley_cat:

**Reply by 201flaviosilva** [reply]
That's exactly it, thank you very much 😊
